pkg/controller/storagecluster/quickstarts.go

```diff
@@ -0,0 +1,39 @@
+package storagecluster
+
+import (
+	"context"
+	"fmt"
+	"github.com/ghodss/yaml"
+	"github.com/go-logr/logr"
+	consolev1 "github.com/openshift/api/console/v1"
+	ocsv1 "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func (r *ReconcileStorageCluster) ensureQuickStarts(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
+	for _, qs := range AllQuickStarts {
+		cqs := consolev1.ConsoleQuickStart{}
+		err := yaml.Unmarshal(qs, &cqs)
+		if err != nil {
+			reqLogger.Info(fmt.Sprintf("failed to unmarshal consolequickstarts, name: %s, namespace: %s", cqs.Name, cqs.Namespace))
```

Comment:
1. cqs is invalid in case Unmarshal fails.
2. Use the preferred logger format instead of fmt.Sprintf.

Reply:
1. will sprint the string which failed the unmarshal if thats ok 
2.changed it.